PharmacyApi/Utilities/ServiceResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PharmacyApi.Utilities
+{
+    public class ServiceResult<T>
+    {
+        public T? Result { get; set; }
+        public bool Success { get; set; }
+        public string? ErrorMessage { get; set; }
+    }
```

Comment:
Interesting - this is a good approach. Can it be enriched w error / success in addition to the overall success flag?

Reply:
Nice to know if its a 500 system error vs a handled Unsuccessful / BadRequest error...